### DISCUSSION TITLE
建议将题目的答案公开

### DISCUSSION THREAD

**Discussion by ziayuan** [AmazingAng/WTF-Solidity#396] [General]
各位管理人员好，我有一个建议，我希望能够放出习题的正确答案，这样方便大家知道错在哪里了。
我觉得大家学习solidity的目的还是能从中学到知识，有些题目的内容在学习内容里面并没有，或者即使看了学习内容，可能个人理解出错，还是不知道正确答案，这种情况下有一个正确答案就很有必要了，我知道管理人员可能怕一些人有了正确答案之后刷分，但我觉得大家既然都花时间来这里学习了，单纯为了刷分的人还是极少数的，不能因为这些人而直接不给出正确答案，这样不方便大家的学习，感谢各位提供的学习材料，希望你们有时间的话能够考虑一下这个建议。

**Comment by ziayuan** [comment]
还有一种情况就是复习的时候有些题忘了正确答案是什么了，这样重新测试还是原来的分数，不方便知道自己到底错没错。

**Reply by gi3636** [reply]
确实是这样，没有准确的答案，根本就不知道错了那里，最起码也告诉我哪题错了

**Comment by AmazingAng** [comment]
因为测试题除了自测，还有考试并领取认证的用途，所以不能公布答案。
如果很多学员有这个需求，我们之后会开放一套有答案的习题用于自测。

**Reply by gi3636** [reply]
好的，实在十分感谢~

**Reply by zjk1314** [reply]
需要需要 没有答案不知道错哪呢 好难受

**Comment by d0m999** [comment]
答案改完以后不更新成绩怎么破？
